pkg/templateservicebroker/brokerapi/templateservicebroker/templateservicebroker.go

```diff
@@ -0,0 +1,26 @@
+package templateservicebroker
+
+import (
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/templateservicebroker/brokerapi"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+type Broker struct {
+	kc *kclientset.Clientset
+	oc *client.Client
+}
+
+func NewBroker(kc *kclientset.Clientset, oc *client.Client) *Broker {
+	return &Broker{kc: kc, oc: oc}
+}
+
+// QUERY: does this need to be unique per cluster?
```

Comment:
no, i'd ask @deads2k if there is already a uuid for the cluster that we can leverage.